specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -0,0 +1,3635 @@
+        }
+      },
+      "get": {
+        "tags": [
+          "Workers"
+        ],
+        "summary": "Retrieves an existing worker by Id",
+        "operationId": "JobRouter_GetWorker",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "workerId",
+            "description": "Id of the worker to retrieve",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/RouterWorker"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a single Worker": {
+            "$ref": "./examples/Workers_GetWorker.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Workers"
+        ],
+        "summary": "Deletes a worker and all of its traces.",
+        "operationId": "JobRouter_DeleteWorker",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "workerId",
+            "description": "Id of the worker to delete",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success"
+          }
+        },
+        "x-ms-examples": {
+          "Delete a Worker and all of its traces": {
+            "$ref": "./examples/Workers_DeleteWorker.json"
+          }
+        }
+      }
+    },
+    "/routing/workers/{workerId}/assignments/{assignmentId}:release": {
+      "post": {
+        "tags": [
+          "Workers"
+        ],
+        "summary": "Releases capacity consumed by an assignment within a workers socket collection.",
+        "operationId": "JobRouter_ReleaseAssignmentAction",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "workerId",
+            "description": "Id of the worker to release assignment capacity",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "assignmentId",
+            "description": "Id of the assignment to release capacity",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "releaseAssignmentRequest",
+            "description": "Request to release assignment",
+            "schema": {
+              "$ref": "#/definitions/ReleaseAssignmentRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
All success responses except 202 & 204 should define a response body.

Reply:
While the body here is not very meaningful currently, we used 200 response inorder to add more parameters to response body later on. This was suggested in one of the meetings by JEFFREYR.